pkg/plugins/policies/meshaccesslog/plugin/v1alpha1/plugin.go

```diff
@@ -24,7 +31,78 @@ func (p plugin) MatchedPolicies(dataplane *core_mesh.DataplaneResource, resource
 	return matchers.MatchedPolicies(api.MeshAccessLogType, dataplane, resources)
 }
 
+func (p plugin) applyToOutbounds(rs *core_xds.ResourceSet, proxy *core_xds.Proxy) error {
+	logPolicies, ok := proxy.Policies.Dynamic[api.MeshAccessLogType]
+	if !ok {
+		return nil
+	}
+
+	sourceService := proxy.Dataplane.Spec.GetIdentifyingService()
+
+	outboundListeners := map[mesh_proto.OutboundInterface]*envoy_listener.Listener{}
+	for _, res := range rs.Resources(envoy_resource.ListenerType) {
+		if res.Origin != generator.OriginOutbound {
+			continue
+		}
+		listener := res.Resource.(*envoy_listener.Listener)
+
+		address := listener.GetAddress().GetSocketAddress()
+
+		outboundListeners[mesh_proto.OutboundInterface{
+			DataplaneIP:   address.GetAddress(),
+			DataplanePort: address.GetPortValue(),
+		}] = listener
+	}
+
+	for _, outbound := range proxy.Dataplane.Spec.Networking.GetOutbound() {
+		oface := proxy.Dataplane.Spec.Networking.ToOutboundInterface(outbound)
+
+		listener, ok := outboundListeners[oface]
+		if !ok {
+			continue
+		}
+
+		serviceName := outbound.GetTagsIncludingLegacy()[mesh_proto.ServiceTag]
+
+		var toPolicy *api.MeshAccessLog_Conf
+		for _, toRule := range logPolicies.ToRules.Rules {
+			if toRule.Subset.IsSubset([]xds.Tag{{
+				Key:   mesh_proto.ServiceTag,
+				Value: serviceName,
+			}}) {
+				toPolicy = toRule.Conf.(*api.MeshAccessLog_Conf)
+				break
+			}
+		}
+		if toPolicy == nil {
```

Comment:
I know there is a similar check in MeshTrafficPermission plugin, but this is not the case anymore. We decided to not include rules with `conf: nil`